UnsupportedPreparedStatementException is not an ERROR in MySQL Proxy

## Feature Request

### Is your feature request related to a problem?

```
[INFO ] 2021-09-09 11:14:56.701 [main] o.a.s.p.i.BootstrapInitializer - Database name is `MySQL`, version is `8.0.25`
[INFO ] 2021-09-09 11:14:56.839 [main] o.a.s.p.frontend.ShardingSphereProxy - ShardingSphere-Proxy start success
[ERROR] 2021-09-09 11:15:00.630 [epollEventLoopGroup-3-104] o.a.s.p.f.c.CommandExecutorTask - Exception occur: 
org.apache.shardingsphere.proxy.frontend.exception.UnsupportedPreparedStatementException: null
	at org.apache.shardingsphere.proxy.frontend.mysql.command.query.binary.prepare.MySQLComStmtPrepareExecutor.execute(MySQLComStmtPrepareExecutor.java:65)
	at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.executeCommand(CommandExecutorTask.java:99)
	at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.run(CommandExecutorTask.java:72)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:384)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

### Describe the feature you would like.

Prepare `begin` is not allowed in MySQL and this situation is under MySQL client's control. This is not an ERROR, so it should not be logged as ERROR.

--- a/shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/exception/ExpectedExceptions.java
+++ b/shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/exception/ExpectedExceptions.java
@@ -42,6 +42,7 @@ public final class ExpectedExceptions {
         EXCEPTIONS.add(SQLParsingException.class);
         EXCEPTIONS.add(CommonDistSQLException.class);
         EXCEPTIONS.add(BackendException.class);
+        EXCEPTIONS.add(UnsupportedPreparedStatementException.class);
     }
     
     /**

--- a/shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/test/java/org/apache/shardingsphere/proxy/frontend/exception/ExpectedExceptionsTest.java
+++ b/shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/test/java/org/apache/shardingsphere/proxy/frontend/exception/ExpectedExceptionsTest.java
@@ -38,6 +38,7 @@ public final class ExpectedExceptionsTest {
         assertTrue(ExpectedExceptions.isExpected(CommonDistSQLException.class));
         assertTrue(ExpectedExceptions.isExpected(BackendException.class));
         assertTrue(ExpectedExceptions.isExpected(NoDatabaseSelectedException.class));
+        assertTrue(ExpectedExceptions.isExpected(UnsupportedPreparedStatementException.class));
     }
     
     @Test

